mathesar_ui/src/sections/import-data/preview/PreviewRows.svelte

```diff
@@ -9,6 +9,7 @@
     FileImport,
     PreviewRow,
   } from '@mathesar/stores/fileImports';
+  import Null from '@mathesar/components/Null.svelte';
```

Comment:
@pavish Is this import okay? Or do we want to add an alias and re-export all components from an `index.ts` file?

Reply:
We have a separate alias for the component library mainly because we want to keep changes to the minimal when we move it out into a separate package.

For mathesar specific components, I don't see a compelling need for an alias. So, I think this import is okay.